config/software/jemalloc.rb

```diff
@@ -1,15 +1,19 @@
 name "jemalloc"
 default_version "4.2.1"
 
+hashes = { "2.2.5" => 'a5c4332705ed0e3fff1ac73cfe975640', "3.6.0" => 'e76665b63a8fddf4c9f26d2fa67afdf2', "4.2.1" => '094b0a7b8c77c464d0dc8f0643fd3901' }
+
 # for td-agent
-version("2.2.5") { source :md5 => 'a5c4332705ed0e3fff1ac73cfe975640' }
-version("3.6.0") { source :md5 => 'e76665b63a8fddf4c9f26d2fa67afdf2' }
-version("4.2.1") { source :md5 => '094b0a7b8c77c464d0dc8f0643fd3901' }
+hashes.each do |var, hash|
```

Comment:
Why not just:
```ruby
version("2.2.5") do
  md5 = 'a5c4332705ed0e3fff1ac73cfe975640'
  source :url => "http://pkgs.fedoraproject.org/repo/pkgs/jemalloc/jemalloc-#{version}.tar.bz2/#{md5}/jemalloc-#{version}.tar.bz2"
         :md5 => md5
end
version("3.6.0") do
  md5 = 'e76665b63a8fddf4c9f26d2fa67afdf2'
  source :url => "http://pkgs.fedoraproject.org/repo/pkgs/jemalloc/jemalloc-#{version}.tar.bz2/#{md5}/jemalloc-#{version}.tar.bz2"
         :md5 => md5
end
version("4.2.1") do
  md5 = '094b0a7b8c77c464d0dc8f0643fd3901'
  source :url => "http://pkgs.fedoraproject.org/repo/pkgs/jemalloc/jemalloc-#{version}.tar.bz2/#{md5}/jemalloc-#{version}.tar.bz2"
         :md5 => md5
end
```
? I know it's a bit more repetitive, but this is configuration, so that should be fine.